get-latest-tgf.sh

```diff
@@ -20,21 +20,21 @@ get_latest_tgf_version () {
 
 script_end () {
     echo 'Done.'
-    tgf --current-version
+    $TGF_PATH/tgf --current-version
     exit 0
 }
 
 install_latest_tgf () {
     VERSION=$(echo $TGF_LATEST_VERSION | cut -d'v' -f 2)
-    
+    TGF_PATH=${TGF_PATH:-/usr/local/bin}
```

Comment:
This line could be deleted.
```suggestion
```

Reply:
It kinda breaks stuff if I delete it because the default path is not set anywhere else

Reply:
No, it's ok. The first line set it `:=` instead of `:-` does the job.

Reply:
So you're saying I could move that line at the top and replace `:=` with `:-` ?

Reply:
I already suggested that change.